### PR TITLE
Functionality added for athletics API

### PR DESCRIPTION
**Pull request by adamdom**
New file to handle athletics API calls. Manually tested on schedules, rosters, searches, and years thoroughly.

### CODE REVIEW

File: server/athletics.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+from flask import jsonify
+from base import *
+from server import app
+from pennathletics import athletes
+from flask import request
+from pennathletics import sportsdata
+from pennathletics import scraper
```
Comment (adelq) [inline server/athletics.py]: `from pennathletics import athletes, sportsdata, scraper`
`from flask import request, jsonify`
Is your Python linter still working?

**Comment by adelq** [comment]
LGTM after PEP8/style fixes

**Comment by adelq** [comment]
Also, you can't say you've "tested throughly" if you haven't written tests

**Comment by dlakata** [comment]
```pennathletics``` should be added to ```requirements.txt```, and ```import server.athletics``` should be added to https://github.com/pennlabs/penn-mobile-server/blob/master/server/__init__.py

**Comment by adamdom** [comment]
All of this should be fixed now

**Comment by adelq** [comment]
It doesn't look like `pennathletics` is in the `requirements.txt`, check out [the current file](https://github.com/pennlabs/penn-mobile-server/blob/36f72b0c5957cebb2dda6c00413b623520d4df23/requirements.txt) for the "syntax"

**Comment by adelq** [comment]
Also, can you add docs for these new routes to the README?

**Pull request merged**